camel-sap/features/org.fusesource.camel.component.sap.feature/pom.xml

```diff
@@ -0,0 +1,85 @@
+<!--
+ Copyright 2013 Red Hat, Inc.
+
+ Red Hat licenses this file to you under the Apache License, version
+ 2.0 (the "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+  implied.  See the License for the specific language governing
+ permissions and limitations under the License.
+ -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.fusesource.components.camel-sap</groupId>
+		<artifactId>features</artifactId>
+		<version>6.3.0.redhat-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>org.fusesource.camel.component.sap.feature</artifactId>
+	<version>6.3.0-SNAPSHOT</version>
+	<packaging>eclipse-feature</packaging>
+
+	<name>JBoss Fuse :: Components :: SAP :: Features :: SAP Data Layer</name>
+	<description>Provides Feature for the SAP Data Layer</description>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.codehaus.mojo</groupId>
+				<artifactId>ianal-maven-plugin</artifactId>
```

Comment:
is the license file a requirement for productization? If yes, maybe we can't just skip it.